block.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 // DiskBlock handles reads against a Block of time series data.
+// XXX: Should we rename "*Block" to "*Segment", as that's what we call these pieces in chunks.go?
```

Comment:
Segment is one of multiple chunk files within a block.
In the end they feel pretty synonymous.  I wouldn't care too much either way. But "block-based" design kinda became the catch phrase for the whole thing already.